docs/modules/ROOT/pages/user-guide/bootstrap.adoc

```diff
@@ -0,0 +1,35 @@
+= Bootstrap Camel Quarkus
+:page-aliases: bootstrap.adoc
+
+Camel Quarkus automatically configure and deploy a Camel Context bean which by default is started/stopped according to the Quarkus Application lifecycle. The configuration step happens at build time during Quarkus' augmentation phase and it is driven by the Camel Quarkus extensions which can be tuned using Camel Quarkus specific properties indentified by "camel.quarkus" as prefix. After the configuration is done, a minimal Camel Runtime is assembled and started at RUNTIME_INIT time. At that point, any additional configuration should be done using CDI, below an example bout how to configure a Camel Component:
```

Comment:
```suggestion
Camel Quarkus automatically configures and deploys a Camel Context bean which by default is started/stopped according to the Quarkus Application lifecycle. The configuration step happens at build time during Quarkus' augmentation phase and it is driven by the Camel Quarkus extensions which can be tuned using Camel Quarkus specific `camel.quarkus.*` properties. After the configuration is done, a minimal Camel Runtime is assembled and started at `RUNTIME_INIT` time. At that point, any additional configuration should be done using CDI. Below is an example showing how to configure a Camel Component:
```

Reply:
I haven't yet marked the `documentation (user guide)` as done,
This is just a bunch of notes and I've committed it as I will continue on another computer so I need a way to synk them :)